package.json

```diff
@@ -43,7 +43,7 @@
     "finalhandler": "1.2.0",
     "fresh": "0.5.2",
     "http-errors": "2.0.0",
-    "merge-descriptors": "1.0.1",
+    "merge-descriptors": "^1.0.3",
```

Comment:
⚠️ If this one merged: 
- #5595 
Should rather use v2:

```suggestion
    "merge-descriptors": "^2.0.0",
```

Reply:
Since we are for sure going to land that, I think we can just land this suggestion.